vcd/resource_vcd_vapp_network.go

```diff
@@ -271,17 +448,92 @@ func resourceVappNetworkDelete(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("error finding vApp: %#v", err)
 	}
 
-	task, err := vapp.RemoveIsolatedNetwork(d.Get("name").(string))
+	_, err = vapp.RemoveNetwork(d.Id())
 	if err != nil {
 		return fmt.Errorf("error removing vApp network: %#v", err)
```

Comment:
It is an error so `%s` should be sufficient instead of `%#v`

Reply:
updated